src/rustdoc-json-types/lib.rs

```diff
@@ -30,7 +30,7 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 /// This integer is incremented with every breaking change to the API,
 /// and is returned along with the JSON blob as [`Crate::format_version`].
 /// Consuming code should assert that this value matches the format version(s) that it supports.
-pub const FORMAT_VERSION: u32 = 46;
+pub const FORMAT_VERSION: u32 = 47;
```

Comment:
If this PR isn't bitrotty or urgent, it'd be really nice if could land after #142601 (and a rebase), so it's guaranteed not to race with #142502 to do the same bump. (#142601 fixes this by making this a conflict, so we'll never need to manually worry about this again 🤞)